tests/integration-tests/tests/update/test_update/test_dynamic_file_systems_update/pcluster.config.update_rollback.yaml

```diff
@@ -0,0 +1,78 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  Iam:
+    S3Access:
+      - BucketName: {{ bucket_name }}
+Scheduling:
+  SlurmSettings:
+    QueueUpdateStrategy: DRAIN
```

Comment:
Change to TERMINATE here

Reply:
I think it would make sense to use TERMINATE in this test because we are focusing on the rollback of a change, so it’s better to test the behaviour with a strategy that would apply a wrong change immediately (TERMINATE) rather than introducing some latency before applying it (DRAIN)